README.md

```diff
@@ -31,6 +31,7 @@ Initial tweet: https://twitter.com/swisscom_csirt/status/1301877750538567680
   | [CyLR](https://github.com/orlikoski/CyLR)                                         | :cloud:                      | :sunny:                                                   | :cloud:                 | :sunny:              | :sunny:                                              | :partly_sunny: <br /> .NET code and re-compilation                                     | :sunny:        | :sunny:                                                 | :cloud:                                                                                                               | :sunny:                                                              | :sunny:                                                 | 
   | [FastIR Collector](https://github.com/SekoiaLab/Fastir_Collector)                             | :cloud:                      | :sunny:                                                   | :partly_sunny:          | :sunny:              | :sunny:                                              | :partly_sunny: <br /> Python code and re-compilation                                   | :cloud:        | :sunny:                                                 | :cloud:                                                                                                               | :scream: <br />last change 3 years old                               | :sunny:                                                 | 
   | [artifactcollector](https://github.com/forensicanalysis/artifactcollector)                            | :cloud:                      | :sunny:                                                   | :sunny:                 | :sunny:              | :sunny:                                              | :partly_sunny: <br /> written in Go, prepare artifacts in YAML ([ForensicArtifacts](https://github.com/forensicartifacts/artifacts))         | :sunny:        | :sunny:                                                 | :cloud:                                                                                                               | :hatching_chick: <br /> young project on Github, only some month old | :partly_sunny:<br />artifactstore                       | 
+  | [CISA Hunt and Incident Response Program (CHIRP) - Archived](https://github.com/cisagov/CHIRP) | ? | :partly_sunny: | :sunny: | :sunny: | :partly_sunny: | :sunny: | :cloud: <br />Require Python and re-compilation |
```

Comment:
Remove the archived addition in the title and move it to column "active development" with adding a cloud and the archived addition there.

Reply:
After studying the project, I would omit having it in the list at all. It's not a live collection tool for forensic artifacts but a tool created to dynamically query Indicators of Compromise (IoCs) on hosts which is a different thing. I didn't find anything related to log or artifact collection, only searching for specific things inside logs or inside registry, ...

Is that correct?

Reply:
I believe so. Removed